### PR TITLE
Next specific URL for push notifications

### PR DESCRIPTION
**Pull request by adgad**
/cc @Financial-Times/next-myft 

No tests yet - but a custom URL for next. Also much repetition, but possibly fine for the interrim.

For a given Concept ID, get me all the endpoints to send push notifications to.

### CODE REVIEW

File: server/controllers/next/endpoints-for-concept.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			res.json({
+				"user": {
+					"uuid": id,
+				},
+				"total": 0,
+				"count": 0,
+				"items": []
+			});
+		}
+		console.log('result', results);
```
Comment (Lrnk) [inline server/controllers/next/endpoints-for-concept.js]: did you mean to leave this log in?

Comment (phamann) [inline server/controllers/next/endpoints-for-concept.js]: Remove debug code.

File: server/controllers/next/endpoints-for-concept.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		since = `WHERE toInt(rel.created) > {since}`;
+	}
+
+	const query = `
+		MATCH(concept:Concept { uuid: "${params.uuid}"})<-[followed]-(u:User)-[rel:enabled]->(e:Endpoint)
+		WITH count(e) AS total, collect(e)[${start}..${end}] AS endpoint, concept
+		RETURN concept, endpoint, total
+	`;
+
+	db.cypher({query, params}).then((results) => {
```
Comment (phamann) [inline server/controllers/next/endpoints-for-concept.js]: Could also pass `lean: true`.

**Comment by phamann** [comment]
:+1: Looks good to me.

**Pull request merged**